torch/_inductor/utils.py

```diff
@@ -1104,8 +1104,20 @@ def _new_line(self, line: str) -> DelayReplaceLine:
 
 @functools.lru_cache(None)
 def is_big_gpu(index) -> bool:
+    prop = torch.cuda.get_device_properties(index)
+
+    # SM logic is not relevant to ROCm gpus
+    # Arbitrarily skipping the older models
+    if torch.version.hip is not None:
```

Comment:
```suggestion
    if torch.version.hip:
```
nit.  the above suggestion is more succinct.